packages/components/src/inputs/src/PasswordField.module.css

```diff
@@ -0,0 +1,33 @@
+.hop-PasswordField  {
+    --hop-PasswordField-gap: var(--hop-space-stack-xs);
+    --hop-PasswordField-default-inline-size: 20rem;
```

Comment:
why is default added to the name? The value doesn't seem to change, there's not another inline-size